src/self-managed-debug/src/k8s_resource_dumper.rs

```diff
@@ -211,6 +228,34 @@ pub async fn dump_namespaced_resources(context: &Context, client: &Client, names
     K8sResourceDumper::<Event>::namespaced(context, client.clone(), namespace.clone())
         .dump()
         .await;
+    K8sResourceDumper::<Materialize>::namespaced(context, client.clone(), namespace.clone())
```

Comment:
It actually blew me away that we could just reuse this struct here and it just works.